lua/entities/gmod_wire_expression2/core/cl_http_cl.lua

```diff
@@ -0,0 +1,131 @@
+--[[
+
+	Clientside HTTP library
+	 - swadical 🍌
```

Comment:
I don't understand: should the comment block be removed? I was following the structure of many other files in the `core/` folder which have: 
 - a title describing the file
 - an attribution to the original author

I understand certain licenses depend on comment blocks at the top of each file.